hack/make/.build-rpm/docker-engine.spec

```diff
@@ -127,13 +127,11 @@ export DOCKER_GITCOMMIT=%{_gitcommit}
 # ./man/md2man-all.sh runs outside the build container (if at all), since we don't have go-md2man here
 
 %check
-./bundles/%{_origversion}/dynbinary-client/docker -v
```

Comment:
Should we just go ahead and remove this packaging stuff since this basically breaks packaging anyway?